data_processors/pipeline/tools/liborca.py

```diff
@@ -173,7 +173,7 @@ def parse_umccrise_workflow_output_directory(output_json: str, deep_check: bool
     :return:
     """
 
-    lookup_keys = ['main/umccrise_output_directory', 'umccrise_output_directory']
+    lookup_keys = ['main/output_directory', 'output_directory']
```

Comment:
@alexiswl Shall we still retain old keys in the lookup list? 

Unlike Pipeline stateful code elsewhere; just for this liborca concern, it is a utility that able to parse backward compatible on older workflow run output. So, that's the goal here I suppose. 

There might be likely chance that we still have to parse older workflow run records in db at some point.

I am thinking like; just simple expanding the list. No other harms, right? What chu reckon? 

```
lookup_keys = ['main/umccrise_output_directory', 'umccrise_output_directory', 'main/output_directory', 'output_directory']
```

Reply:
Oh right, yes that makes sense

Reply:
Done in d6014ef4755c2e42753e27c383cbb776766bd917

Reply:
Ah. Almost. Typo there in old key; the second one without main prefix. 🙂